packages/features/ee/payments/components/Payment.tsx

```diff
@@ -85,16 +82,18 @@ export default function PaymentComponent(props: Props) {
     };
     if (paymentOption === "HOLD" && "setupIntent" in props.payment.data) {
       const setupIntentData = props.payment.data as unknown as StripeSetupIntentData;
-      payload = await stripe.confirmCardSetup(setupIntentData.setupIntent.client_secret!, {
-        payment_method: {
-          card,
+      payload = await stripe.confirmSetup({
+        elements,
+        confirmParams: {
+          return_url: `${CAL_URL}/booking/${props.bookingUid}`,
```

Comment:
@hariombalhara the confirm params with the return URL is required. Do we still need the `bookingSuccessRedirect` function?